rasa/core/run.py

```diff
@@ -121,7 +121,7 @@ def configure_app(
         rasa.core.utils.list_routes(app)
 
     # configure async loop logging
```

Comment:
not your change, but I think that comment is useless (the method name says exactly the same)
```suggestion
```